docs/guides/tutorials/getting_started/create_your_first_expectations.rst

```diff
@@ -3,44 +3,67 @@
+
+.. code-block:: bash
+
+    Heads up! This feature is Experimental. It may change. Please give us your feedback!
+
+    Which table would you like to use? (Choose one)
+        1. yellow_tripdata_sample_2019_01 (table)
+        2. yellow_tripdata_staging (table)
+        Do not see the table in the list above? Just type the SQL query
     : 1
-    
-    Name the new Expectation Suite [npidata_pfile_20200511-20200517.warning]: 
-    
-    Great Expectations will choose a couple of columns and generate expectations about them
-    to demonstrate some examples of assertions you can make about your data.
-    
-    Great Expectations will store these expectations in a new Expectation Suite 'npidata_pfile_20200511-20200517.warning' here:
-    
-      file:///home/ubuntu/example_project/great_expectations/expectations/npidata_pfile_20200511-20200517/warning.json
-    
-    Would you like to proceed? [Y/n]: 
-    
-    Generating example Expectation Suite...
-    
-    Done generating example Expectation Suite
+
+
+Which data should I choose when creating Expectations?
+====================
+
+In this example, we want to build an Expectation Suite based on what we know about our taxi data: each taxi ride has a ``passenger_count`` between 1 and 6. We then want to use that Expectation Suite to validate any future data that is loaded into the ``staging`` table. Hence, we choose the ``yellow_tripdata_sample_2019_01`` table when creating the new Expectation Suite, which we will then use to validate the ``yellow_tripdata_staging`` table **in a later step**. Makes sense, right?
+
+After selecting the table, Great Expectations will open a Jupyter notebook which will take you through the next part of the ``scaffold`` workflow.
+
+
+Creating Expectations in Jupyter notebooks
+-------------------
+
+In this section, we will explain how to create and edit Expectations in a Jupyter notebook. Notebooks are a simple way of interacting with the Great Expectations Python API. You could also just write all this in plain Python code, but for convenience, Great Expectations provides you some boilerplate code in notebooks.
+
+Here's what's happening in the notebook:
+
+#. The first cell does several things: It imports all the relevant libraries, loads a Data Context, and creates what we call a Batch of your data and Expectation Suite.
+
+#. The second cell allows you to specify which columns you want to run the automated profiler on. Remember how we want to add some tests on the ``passenger_count`` column to ensure that its values range between 1 and 6? **Let's uncomment just that one line:**
+
+.. code-block:: python
+
+    included_columns = [
+        # 'vendor_id',
+        # 'pickup_datetime',
+        # 'dropoff_datetime',
+        'passenger_count',
+        ...
+    ]
+
+#. The next cell passes the scaffolder config to the ``BasicSuiteBuilderProfiler``, which will then profile the data and create the relevant Expectations to add to your ``taxi.demo`` suite.
```

Comment:
I think "profiler" config is better here, since I don't think "scaffolder" is really precisely used (i.e. it's just a way of using a profiler or even intent for using a profiler)